src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/controllers/AttachExceptionRecordToExistingCaseTest.kt

```diff
@@ -62,6 +63,11 @@ class AttachExceptionRecordToExistingCaseTest {
 
     private val wireMock by lazy { WireMock(wireMockPort) }
 
+    private val startEvent = get(
+        "/caseworkers/640/jurisdictions/BULKSCAN/case-types/Bulk_Scanned"
+            + "/cases/1539007368674134/event-triggers/attachScannedDocs/token"
```

Comment:
can any constants be used?
Also, what's up with hardcoded case id?

Reply:
This is all static and hard coded already. There is not much point in making it dynamic right now. Maybe in a later PR.